src/common-components/tests/SocialAuthProviders.test.jsx

```diff
@@ -18,6 +18,24 @@ describe('SocialAuthProviders', () => {
     loginUrl: '/auth/login/apple-id/?auth_entry=login&next=/dashboard',
   };
 
+  it('should match social auth providers alignment snapshot', () => {
```

Comment:
I guess we have decided not to write any snapshot tests further. You can remove this because it's not providing much value and not ensuring the new alignment. If someone changes the code in the future and the alignment isn't correct, they will simply update the snapshot and the test will pass again.